autoscheduler/frontend/src/App.tsx

```diff
@@ -1,25 +1,22 @@
 import * as React from 'react';
 import { Router, Link } from '@reach/router';
-import { AppBar, IconButton, Toolbar } from '@material-ui/core';
+import { AppBar, IconButton, Toolbar, Button } from '@material-ui/core';
 import HomeIcon from '@material-ui/icons/Home';
 import { ThemeProvider } from '@material-ui/styles';
 
 import theme from './theme';
 import Empty from './components/Empty';
+import NavBar from './Components/NavBar';
```

Comment:
`./components` needs to be lower-case in Linux environments, such as my WSL and Docker